proxy/src/main/java/com/wavefront/agent/PointHandlerImpl.java

```diff
@@ -172,7 +172,8 @@ public void handleBlockedPoint(String pointLine) {
     randomPostTask.incrementBlockedPoints();
   }
 
-  private static final long MILLIS_IN_YEAR = DateUtils.MILLIS_PER_DAY * 365;
+  //private static final long MILLIS_IN_YEAR = DateUtils.MILLIS_PER_DAY * 365;
+  private static final long MILLIS_DISCARD = DateUtils.MILLIS_PER_DAY * (PushAgent.discardPoints/24);
```

Comment:
You can't really do this since it depends on initialization order. you probably need to plop this in from the top